benchmark_runner/workloads/vdbench_pod.py

```diff
@@ -95,14 +97,10 @@ def run(self):
                     self._oc.wait_for_initialized(label=f'app={name}', label_uuid=False)
                     self._oc.wait_for_ready(label=f'app={name}', label_uuid=False)
                 proc = []
-                scale = int(self._scale)
-                count = 0
-                for scale_node in range(len(self._scale_node_list)):
-                    for scale_num in range(scale):
-                        count += 1
-                        p = Process(target=self.__run_scale, args=(str(count),))
-                        p.start()
-                        proc.append(p)
+                for vm_num in range(len(self._scale_node_list) * self.__scale):
+                    p = Process(target=self.__run_scale, args=(str(vm_num+1),))
+                    p.start()
+                    proc.append(p)
```

Comment:
You might want to reap any processes that are already complete, so you don't have a lot of zombie processes around.  I don't know the Process stuff well enough to know whether you can do that.